benchmarks/flame_graph_loop.js

```diff
@@ -23,10 +23,10 @@ var InMemoryReporter = require('./dist/src/reporters/in_memory_reporter.js').def
 var Tracer = require('./dist/src/tracer.js').default;
 var opentracing = require('opentracing');
 
-function server_client_span_loop(tracer, carrier) {
+function server_client_span_loop(tracer) {
     for ( var i = 0; i <= 0; i--) {
         // deserialize parent context from carrier
-        var parentContext = tracer.extract(opentracing.FORMAT_HTTP_HEADERS, carrier);
+        var parentContext = tracer.extract(opentracing.FORMAT_HTTP_HEADERS, {});
```

Comment:
yeah but, the way I coded it is a bit strange, because I want context to be null.  So I could have just emitted this line.